src/main/java/com/java_academy/ServerApp.java

```diff
@@ -21,13 +26,21 @@
     public static void main(String[] args) {
         try {
             ServerSocket serverSocket = new ServerSocket();
-            SocketProvider socketProvider = new ServerSocketProvider(serverSocket, messageSupplier -> {
-                System.out.println("message from a Client: " + messageSupplier.get());
-            });
+            SocketProvider socketProvider = new ServerSocketProvider(serverSocket);
 
             InetSocketAddress inetSocketAddress = new InetSocketAddress("localhost", 3000);
```

Comment:
I know it's not added change but you'd rather want those to be static constants.